### DISCUSSION TITLE
Mircr presets different from documentation

### DISCUSSION THREAD

**Discussion by alvin-MGI** [milaboratory/mixcr#1389] [Q&A] [answered]
Dear Mixcr team,

This list of presets that i see when i type the command "mixcr listPresets" is different from what i see in the documentation.

For example, I'm interested to run the Takara library profiling given in https://mixcr.com/mixcr/guides/takara-hsa-bcr/
The preset given is **takara-human-rna-bcr-umi-smartseq**
However, "mixcr listPresets" gives
**takara-human-bcr-cdr3
takara-human-bcr-full-length
takara-human-tcr-V1-cdr3
takara-human-tcr-V1-full-length
takara-human-tcr-V2-cdr3
takara-human-tcr-V2-full-length**

Are you able to tell me what is the difference in the 2 presets? The "listPresets" seems to be more specific in nature. 

How do i obtain the results from  **takara-human-rna-bcr-umi-smartseq** with the different takara presets above?

Best Regards,
Alvin

**Comment by mizraelson** [comment, marked as answer]
Hi,
The presets in the documentation are aligned with the latest MiXCR v4.5. What version do you use?

Sincerely,
Mark
